doc/source/whatsnew/v1.0.0.rst

```diff
@@ -438,6 +438,7 @@ Groupby/resample/rolling
 - Bug in :meth:`DataFrameGroupby.agg` not able to use lambda function with named aggregation (:issue:`27519`)
 - Bug in :meth:`DataFrame.groupby` losing column name information when grouping by a categorical column (:issue:`28787`)
 - Bug in :meth:`DataFrameGroupBy.rolling().quantile()` ignoring ``interpolation`` keyword argument (:issue:`28779`)
+- Bug in :meth:`SeriesGroupBy.count`, :meth:`SeriesGroupBy.size`, :meth:`SeriesGroupBy.nunique` and :meth:`SeriesGroupBy.nth` missing unobserved categories when ``observed=False`` (:issue:`17605`)
```

Comment:
can you make a sub-section in the api-changing section. this needs an example of the current and changed behavior as its non-trivial what has changed (you can use the example from the OP).